docs/de/docs/tutorial/first-steps.md

```diff
@@ -0,0 +1,335 @@
+
+### Schritt 2: Erstellen einer `FastAPI` "Instanz"
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Hier wird die Variable `app` eine `Instanz` der Klasse `FastAPI` sein.
+
+Dies wird der Hauptpunkt der Interaktion sein, um Ihre gesamte API zu erstellen.
+
+Diese "App" ist dieselbe, auf die im Befehl "uvicorn" verwiesen wird:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+Wenn Sie Ihre App so erstellen:
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial002.py!}
+```
+
+Und in eine Datei `main.py` schreiben, würden Sie `uvicorn` wie folgt aufrufen:
+
+<div class="termy">
+
+```console
+$ uvicorn main:my_awesome_api --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+### Schritt 3: Erstellen einer *Pfadoperation*
+
+#### Pfad
+
+"Pfad" bezieht sich hier auf den letzten Teil der URL, beginnend mit dem ersten `/`.
+
+Also für eine URL wie:
+
+```
+https://example.com/items/foo
+```
+
+...wäre der Pfad:
+
+```
+/items/foo
+```
+
+!!! Information
+    Ein "Pfad" wird gemeinhin auch als "Endpunkt" oder "Route" bezeichnet.
+
+Bei der Erstellung einer API ist der "Pfad" die wichtigste Methode zur Trennung von "Anliegen" und "Ressourcen".
+
+#### Operation
+
+"Operation" bezieht sich hier auf eine der HTTP-"Methoden".
+
+wie:
+
+* `POST`
+* `GET`
+* `PUT`
+* `DELETE`
+
+...und die exotischeren unter ihnen:
+
+* `OPTIONS`
+* `HEAD`
+* `PATCH`
+* `TRACE`
+
+Beim HTTP-Protokoll können Sie mit jedem Pfad über eine (oder mehrere) dieser "Methoden" kommunizieren.
+
+---
+
+Bei der Erstellung von APIs verwenden Sie normalerweise diese spezifischen HTTP-Methoden, um eine bestimmte Aktion durchzuführen.
+
+Normalerweise verwenden Sie:
+
+* `POST`: um Daten anzulegen.
+* `GET`: um Daten zu lesen.
+* `PUT`: um Daten zu aktualisieren.
+* `DELETE`: um Daten zu löschen.
+
+In der OpenAPI wird also jede der HTTP-Methoden als "Operation" bezeichnet.
+
+Wir werden sie auch "**Operationen**" nennen.
+
+#### Definieren eines *Pfadoperations-Dekorators*.
+
+```Python hl_lines="6"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Das `@app.get("/")` sagt **FastAPI**, dass die Funktion direkt darunter für die Bearbeitung von Anfragen zuständig ist, die an:
+
+* den Pfad `/`
+* mit einer <abbr title=" HTTP GET Methode"><code>get</code> Operation</abbr> gesendet werden.
+
+!!! Information "`@decorator` Info"
+
+
+    Diese `@etwas` Syntax wird in Python "Decorator" genannt.
+
+    Man setzt ihn über eine Funktion. Wie ein hübsches dekoratives Hütchen (daher kommt wohl der Begriff).
+
+    Ein "Dekorator" nimmt die darunter stehende Funktion und erweitert sie.
```

Comment:
Ein "Dekorator" erweitert die darunter stehende Funktion. 

That's it?

Reply:
yes, better